README.md

```diff
@@ -1,210 +1,305 @@
+the <kbd>NumRow</kbd> layer:
+
+- all digits are on the home row, in the order you already know
+- the upper row helps with <kbd>Shift</kbd>-digit shortcuts
+- the lower row has dash, comma, dot and slash signs to help with number / date
+inputs
+- <kbd>Space</kbd> becomes a narrow no-break space for layouts that supports it
+
+![NumRow layer on a 33-key keyboard](./img/numrow.svg)
 
-![base, navigation and sym layers on a 33-key keyboard](img/selenium33/all.svg)
+Even on keyboards that *do* have a physical number row, this `NumRow`layer can
+be interesting to use in order to minimize finger movements furthermore.
 
-It uses 4 layers (instead of 3 for Arsenik), which makes it a natural fit
-for 34-key keyboards like the [Ferris][34].
 
-- Vim-like navigation in all apps, with any OS layout
+### 4. Navigation layer
+
+A basic <kbd>Navigation</kbd> layer has an arrow cluster on the left hand to
+move around and a num pad on the right hand.
+
+![navigation layer on a 33-key keyboard](./img/navigation.svg)
+
+#### A superpowered Vim-friendly mod
+
+For those who like to move the cursor with <kbd>HJKL</kbd> in all app, with any
+keyboard layout, it is possible to enable a Vim-like <kbd>Navigation</kbd>
+layer.
+
+It also has:
 - super-comfortable <kbd>Tab</kbd> and <kbd>Shift</kbd>-<kbd>Tab</kbd>
 - mouse emulation: previous / next and mouse scroll
-- easy left-hand shortcuts
 
-![Vim navigation layer on a 33-key keyboard](img/selenium33/navigation.svg)
+![Vim navigation layer on a 33-key keyboard](./img/vim_navigation.svg)
 
 This <kbd>Navigation</kbd> layer has a few empty slots on purpose, so you can
 add our own keys or layers.
 
+<kbd>NumPad</kbd> and <kbd>Fn</kbd> toggle these layers, they stay active
+without holding the key until escaped with <kbd>Alt</kbd> or <kbd>AltGr</kbd>.
 
-#### NumRow >> NumPad
+![NumPad layer on a 33-key keyboard](./img/numpad.svg)
+<p align="center">
+  <em>NumPad layer toggled</em>
+</p>
 
-In <kbd>Symbol</kbd> mode, pressing the left thumb key brings up the
-<kbd>NumRow</kbd> layer:
+![Fn layer on a 33-key keyboard](./img/fn.svg)
+<p align="center">
+  <em>Fn layer toggled</em>
+</p>
 
-- all digits are on the home row, in the order you already know
-- the upper row helps with <kbd>Shift</kbd>-digit shortcuts
-- the lower row has dash, comma, dot and slash signs to help with number / date
-inputs
+### 5. Keyboard layout
 
-![NumRow layer on a 33-key keyboard](img/selenium33/numrow.svg)
+Choose your keyboard layout among the available ones for Arsenik to work
+properly.
 
-Even on keyboards that *do* have a physical number row, this <kbd>NumRow</kbd>
-layer can be interesting to use in order to minimize finger movements further
-more. And it makes it easier to mix symbols with numbers (e.g. `[0]`).
+If your layout is not on this list, feel free to open an issue or upvote an
+existing one.
 
+Here is some specifities for some supported layouts:
```

Comment:
I think we should mention here that Arsenik works out-of-the-box with Lafayette layouts because their AltGr layer already matches Arsenik’s Symbols layer. And Qwerty/Colemak should work too with the same Symbols layer.

The way it is now, it kinda lets me think Arsenik has been designed for Azerty first.

Reply:
Agreed, I’ve added `Ergo‑L/Qwerty‑Lafayette/other Lafayette layouts` and `Qwerty/Colemak` blocks.